ci.jsonnet

```diff
@@ -416,15 +416,11 @@ local part_definitions = {
     test_integration: { run+: jt(["test", "integration"]) },
     test_gems: { run+: jt(["test", "gems"]) },
     test_ecosystem: { run+: jt(["test", "ecosystem"]) },
-    test_bundle: { run+: jt(["test", "bundle", "--no-sulong"]) },
+    test_bundle: { run+: jt(["test", "bundle"]) },
```

Comment:
Sorry I meant to remove the line completely.

Reply:
Ah, because it is already run as part of `test_cexts`, right.